main.tf

```diff
@@ -67,22 +66,25 @@ resource "google_compute_instance_template" "default" {
   }
 }
 
+provider "google-beta" {
```

Comment:
Tried to use your PR, but faced with the issue where I need to provide additional options to the provider. Currently,  It's no possible. 

Maybe better way do not put providers to the modules, just left notice to the README, so we have to define both providers (stable and beta).

Reply:
Sorry about that, it's generally a bad idea to put providers in a module for just this sort of reason, I was playing about with it and forgot about it. Fixed now